pom.xml

```diff
@@ -1,134 +1,118 @@
-  <description>Uploads iOS, Mac or Android apps to a HockeyApp server.</description>
-  <version>1.2.3-SNAPSHOT</version>
-  <packaging>hpi</packaging>
-
- <licenses>
-     <license>
-         <name>MIT license</name>
-         <comments>All source code is under the MIT license.</comments>
-     </license>
- </licenses>
-
-  <!-- get every artifact through repo.jenkins-ci.org, which proxies all the artifacts that we need -->
-  <repositories>
-    <repository>
-      <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
-    </repository>
-  </repositories>
-
-  <scm>
-    <connection>scm:git:ssh://github.com/jenkinsci/hockeyapp-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/hockeyapp-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/hockeyapp-plugin</url>
-    <tag>HEAD</tag>
-  </scm>
-
-  <distributionManagement>
-    <repository>
-      <id>maven.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/releases/</url>
-    </repository>
-    <snapshotRepository>
-      <id>maven.jenkins-ci.org</id>
-      <url>https://repo.jenkins-ci.org/snapshots/</url>
-    </snapshotRepository>
-  </distributionManagement>
-
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/HockeyApp+Plugin</url>
-  <developers>
-    <developer>
-      <id>ohoeltke</id>
-      <name>Oliver Hoeltke</name>
-      <email>ohoeltke@gmail.com</email>
-    </developer>
-    <developer>
-      <id>Brantone</id>
-      <name>Brenton Bartel</name>
-    </developer>
-  </developers>
-  <pluginRepositories>
-    <pluginRepository>
-      <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
-    </pluginRepository>
-  </pluginRepositories>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
-        <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5.1</version>
-        <dependencies>
-          <dependency>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>3.4</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>io.jenkins.plugins</groupId>
+
+    <artifactId>hockeyapp</artifactId>
+
+    <version>1.2.3-SNAPSHOT</version>
+
+    <packaging>hpi</packaging>
+
+    <properties>
+        <jenkins.version>1.609.2</jenkins.version>
```

Comment:
If you require Java 7, it's better to update to Jenkins 1.625.3. Just because 1.609.x can run on Java 6 and hence the plugin may not work there